package.json

```diff
@@ -4,12 +4,55 @@
   "description": "A library for simplyfing the use of redux-thunk with REST APIs",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha --require babel-polyfill --compilers js:babel-register src/*_test.js",
+    "watch": "mocha --watch --require babel-polyfill --compilers js:babel-register src/*_test.js",
+    "lint": "standard --env mocha",
+    "flow": "flow",
+    "fmt": "standard --env mocha --fix"
   },
   "keywords": [
     "redux",
     "REST"
   ],
   "author": "Alice Pote <alicep@mit.edu>",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "babel-polyfill": "^6.23.0",
+    "lodash": "^4.17.4",
```

Comment:
Not necessary for this PR but it would be nice to get rid of this dependency and switch entirely to ramda, to reduce the JS bundle size for end users. It looks like there's only three things we need to change, not sure if they're trivial or not